packages/eslint-scope/Makefile.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * @fileoverview Build file
+ * @author nzakas
+ * @copyright jQuery Foundation and other contributors, https://jquery.org/
```

Comment:
jquery foundation has been renamed to openjs foundation - https://openjsf.org/

it's not a blocker - we may want to update all these info in a following pr.